openms.rb

```diff
@@ -0,0 +1,45 @@
+class Openms < Formula
+  desc "Open-source software C++ library for LC-MS data management & analyses"
+  homepage "http://www.openms.de"
+  url "https://github.com/OpenMS/OpenMS/releases/download/Release2.2.0/OpenMS-2.2.0-src.tar.gz"
+  version "2.2"
+  sha256 "61e7d24890abb6f462e2f026efb0dc88b84639b228d96e919c8074d3b11d286e"
+
+## Adding the images to be skipped while testing 
+# KAUST_SKIP centos:6 centos:7
+
+  depends_on "cmake" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "boost"
+  depends_on "glpk"
+  depends_on "libsvm"
+  depends_on "bzip2"
+  depends_on "homebrew/science/coinmp"
+  depends_on "eigen"
+
+  def install
+    ENV.prepend_create_path "PATH", "/usr/lib64/qt4/bin" if File.exist?("/etc/redhat-release")
```

Comment:
@asmaahassan90  do we need the part containing redhat-release for Ubuntu or just CentOS ?

Reply:
it was added to check if OS is centos or redhat to set qt path properly. but we might not need it right now. Will edit the formula and set it to default